mathsys/dev/ir/local.py

```
@@ -0,0 +1,72 @@
+#> HEAD -> MODULES
+from typing import cast
+
+
+#^
+#^  TYPES
+#^
+
+#> TYPES -> U8 CLASS
+class u8:
+    def __new__(cls, value: int) -> bytes:
+        if not 1 <= value <= 2**8 - 1: raise ValueError(f"'{value}' is outside range for u8.")
```

Ruff (0.14.8)

Multiple statements on one line (colon)

(E701)